src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -2235,7 +2235,7 @@ void GameObject::BuildValuesUpdate(uint8 updateType, ByteBuffer* data, Player* t
 
             if (index == GAMEOBJECT_DYNAMIC)
             {
-                uint16 dynFlags = 0;
+                uint16 dynFlags = m_uint32Values[GAMEOBJECT_DYNAMIC];
```

Comment:
so I guess u have to be a collab to make this type of changes so the rest wont go all retard about mixed changes #TClogic